_includes/sub-template-prerequisites-storage.md

```diff
@@ -0,0 +1,11 @@
+### Azure Storage account 
+
+If you are deploying a Java EE application or using your own JDBC datasource driver, you are required to 
```

Comment:
datasource  -> data source (two words)

Reply:
Fixed in https://github.com/oracle/weblogic-azure/pull/88/commits/09c58c430464aa1f8ba8a8f2e0efc00dc141e6fb